depPool.go

```diff
@@ -188,7 +188,7 @@ func (dp *depPool) findProvides(pkgs types.StringSet) error {
 		words := strings.Split(pkg, "-")
 
 		for i := range words {
-			results, err = rpc.SearchByNameDesc(strings.Join(words[:i+1], "-"))
+			results, err = rpc.Search(strings.Join(words[:i+1], "-"))
```

Comment:
In depPool wouldn't it be better to search only Name instead of NameDesc?

Reply:
I see no reason to. More results means a better chance of finding a provide. And seems as we can fit everything into a single into query there should be no speed difference.

Reply:
Ok, I can agree with that.